src/vanilla/store.ts

```diff
@@ -199,14 +199,27 @@ const addPendingFunction = (pending: Pending, fn: () => void) => {
 }
 
 const flushPending = (pending: Pending) => {
+  let error: unknown | undefined
+  const call = (fn: () => void) => {
+    try {
+      fn()
+    } catch (e) {
+      if (!error) {
+        error = e
+      }
```

Comment:
Only throw the first error, right?

Reply:
That is correct.

I wish there was some way to throw the next error after the currently thrown error was caught. There isn't.